zap/selenium/ci/TypeScript/test/front_guest/product_detail.test.ts

```diff
@@ -0,0 +1,41 @@
+import { test, expect, chromium, Page } from '@playwright/test';
+import { intervalRepeater } from '../../utils/Progress';
+import { ZapClient, Mode, ContextType, Risk, HttpMessage } from '../../utils/ZapClient';
+const zapClient = new ZapClient('http://127.0.0.1:8090');
+
+const baseURL = 'https://ec-cube';
+const url = baseURL + '/products/detail/2';
+
+test.describe.serial('商品詳細画面のテストをします', () => {
+  let page: Page;
+  test.beforeAll(async () => {
+    await zapClient.setMode(Mode.Protect);
+    await zapClient.newSession('/zap/wrk/sessions/front_guest_contact', true);
```

Comment:
```suggestion
    await zapClient.newSession('/zap/wrk/sessions/front_guest_product_detail', true);
```
セッションのファイル名も変更お願いします